pdns/recursordist/test-syncres_cc.cc

```diff
@@ -140,12 +140,7 @@ void initSR(bool debug)
     g_log.setLoglevel(Logger::None);
     g_log.toConsole(Logger::Error);
   }
-
-  RecursorPacketCache::s_refresh_ttlperc = 0;
-  MemRecursorCache::s_maxServedStaleExtensions = 0;
-  MemRecursorCache::s_maxRRSetSize = 100;
-  MemRecursorCache::s_limitQTypeAny = true;
-  NegCache::s_maxServedStaleExtensions = 0;
+  MemRecursorCache::resetStaticsForTests();
```

Comment:
Unless I'm mistaken, one line:
```c++
 NegCache::s_maxServedStaleExtensions = 0;
```
has been lost in this change, is that OK?

Reply:
Oops, fixed